src/main/java/core/basesyntax/service/DailyReport.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax.service;
+
+import core.basesyntax.strategy.OperationStrategy;
+import core.basesyntax.strategy.OperationStrategyImpl;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class DailyReport {
+    private final List<String> result = new ArrayList<>();
+    private final Map<String, Integer> fruitMap = new HashMap<>();
```

Comment:
lets create separate class Storage, where we will store information about Fruits quantity